server/mediaserver.go

```diff
@@ -576,7 +562,24 @@ func (s *LivepeerServer) HandlePush(w http.ResponseWriter, r *http.Request) {
 		st := stream.NewBasicRTMPVideoStream(appData)
 		cxn, err = s.registerConnection(st)
 		// TODO monitor for activity and clean up
+
+		ticker := getConnectionTicker()
+
+		go func(s *LivepeerServer, mid core.ManifestID) {
```

Comment:
Minimizing free variables isn't strictly necessary here, but I like it anyway :+1: